### PR TITLE
[Doc] DOCT-13264 - details about icons + compatibility information

### PR DESCRIPTION
**Pull request by florentlb**
### Requirements

* Any code change adding any logic **MUST** be tested through a unit test executed with the default build
* Any API addition **MUST** be done with a documentation update if relevant

### CODE REVIEW

File: documentation/src/main/antora/modules/ROOT/pages/studio-from-car.adoc [hunk 1]
```diff
@@ -57,3 +57,15 @@ Make sure you are connected to a remote project and the artifact repository for
...
+
+
+== Troubleshooting
+
+*My custom component builds correctly but does not appear in Talend Studio, how to fix it?*
+This issue can be caused by the icon specified in the component metadata.
+
+* Make sure to specify a custom icon for the component and the component family.
+* These custom icons must be in PNG format to be properly handled by Talend Studio.
+* Remove SVG parameters from the `talend.component.server.icon.paths` property in the HTTP server configuration.
```
Comment (undx) [inline documentation/src/main/antora/modules/ROOT/pages/studio-from-car.adoc]: Perhaps, put a link to the relevant place to do that?

File: documentation/src/main/antora/modules/ROOT/pages/component-registering.adoc [hunk 2]
```diff
@@ -46,8 +47,7 @@ When you generate your project and import it in your IDE, icon and version both
 
 - *@Icon*: Sets an icon key used to represent the component.
 You can use a custom key with the `custom()` method but the icon may not be rendered properly. The icon defaults to *Star*. +
```
Comment (undx) [inline documentation/src/main/antora/modules/ROOT/pages/component-registering.adoc]: it was changed to check in TCOMP-1612.

Comment (florentlb) [reply documentation/src/main/antora/modules/ROOT/pages/component-registering.adoc]: updated

File: documentation/src/main/antora/modules/ROOT/pages/studio-from-car.adoc (hunk 1 above)
Comment (florentlb) [reply documentation/src/main/antora/modules/ROOT/pages/studio-from-car.adoc]: added a link to the list of configs

**Pull request merged**